src/pydata_sphinx_theme/translator.py

```diff
@@ -81,9 +81,17 @@ def visit_table(self, node):
         if "align" in node:
             classes.append(f'table-{node["align"]}')
 
+        # put table within a scrollable container because overflow on display: table does nothing
```

Comment:
It's actually the width property that doesn't really do anything other than [act as a kind of minimum width for the table](https://www.w3.org/TR/CSS22/tables.html#auto-table-layout)